parm/fixed_files_mapping.yaml

```diff
@@ -2,200 +2,35 @@ fixed_files:
   #
   #-----------------------------------------------------------------------
   #
-  # FV3_NML_VARNAME_TO_SFC_CLIMO_FIELD_MAPPING:
-  # This array is used to set some of the namelist variables in the forecast 
-  # model's namelist file that represent the relative or absolute paths of 
-  # various fixed files (the first column of the array, where columns are 
-  # delineated by the pipe symbol "|") to the full paths to surface climatology 
-  # files (on the native FV3-LAM grid) in the FIXlam directory derived from 
-  # the corresponding surface climatology fields (the second column of the 
-  # array).
+  # These are the names of the fields generated by the make_sfc_climo
+  # task.
```

Comment:
Yes. They do eventually get added to the `files_to_link` section of the fv3 UW YAML. Instead of listing the names of CCPP Suites that use Thompson MP, though, SRW calls a function that loads the XML to find out if Thompson is used. I could definitely clean up this section and move it over to CCPP suites to put it closer to its other relevant settings.